main.js

```diff
@@ -16,17 +20,19 @@ socket.on('connect', function() {
 	 * replacing this line with something that instead supplies the user_id via an environment variable, e.g.
 	 * var user_id = process.env.BOT_USER_ID;
 	 */
-	var user_id = 'my_example_bot_id';
-	var username = 'Example Bot';
-
+	var user_id = 'my_example_tr_bot'; // CHANGE ME
+        // Bot usernames must start with [Bot] and be no longer than 18 characters
+	var username = '[Bot]ExampleTRBot'; // CHANGE ME
 	// Set the username for the bot.
 	// This should only ever be done once. See the API reference for more details.
-	socket.emit('set_username', user_id, username);
+	// socket.emit('set_username', user_id, username);
 
 	// Join a custom game and force start immediately.
 	// Custom games are a great way to test your bot while you develop it because you can play against your bot!
-	var custom_game_id = 'my_private_game';
+	var custom_game_id = 'my_private_gameTR';
```

Comment:
I doubt they will like you having your initials here. But maybe you weren't trying to get it merged.

Reply:
Thanks for pointing that out.